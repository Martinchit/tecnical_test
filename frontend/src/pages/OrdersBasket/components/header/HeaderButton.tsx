import styled from 'styled-components';
import { Button } from 'react-bootstrap';

export const HeaderButton = styled(Button)`
  margin-left: 2%;
  width: 10%;

  @media (max-width: 767px) {
    width: 45%;
    margin-top: 3%;
    font-size: 0.8rem;
  }

  &:focus {
    outline: none;
    box-shadow: none;
  }
`;
