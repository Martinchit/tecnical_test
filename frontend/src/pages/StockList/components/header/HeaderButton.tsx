import styled from 'styled-components';
import { Button } from 'react-bootstrap';

export const HeaderButton = styled(Button)`
  margin-left: 1%;
  width: 10%;
  min-width: 85px;

  @media (max-width: 767px) {
    width: 50%;
    margin-top: 5%;
    font-size: 0.8rem;
  }

  &:focus {
    outline: none;
    box-shadow: none;
  }
`;
