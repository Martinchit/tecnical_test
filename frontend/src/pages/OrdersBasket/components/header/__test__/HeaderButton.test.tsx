import React from 'react';
import { shallow } from 'enzyme';

import { HeaderButton } from '../HeaderButton';

describe('HeaderButton', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeaderButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
