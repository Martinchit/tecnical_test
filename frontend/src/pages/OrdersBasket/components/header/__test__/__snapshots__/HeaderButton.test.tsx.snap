// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderButton renders correctly 1`] = `
<Button
  active={false}
  className="sc-AxjAm dhIUPC"
  disabled={false}
  type="button"
  variant="primary"
/>
`;
